internal/elasticsearch/logstash/pipeline.go

```diff
@@ -71,7 +71,7 @@ func ResourceLogstashPipeline() *schema.Resource {
 			Optional:    true,
 			Elem: &schema.Schema{
 				Type:    schema.TypeString,
-				Default: nil,
+				Default: "{}",
```

Comment:
This sets a default value for individual metadata keys, not the entire metadata map. You likely want to define `DiffSuppressFunc` on the actual `pipeline_metadata` schema object.